frontend/src/pages/EmployeePage.tsx

```diff
@@ -12,24 +12,26 @@ import "../styles/InteractionsPane.css";
 import "../styles/EmployeeDashboard.css";
 import config from "../../config.json";
 
-interface EmployeePageProps {
-  employeeHID: number;
-}
 
-export default function EmployeePage({ employeeHID }: EmployeePageProps) {
+export default function EmployeePage() {
+
+  const [EmployeeHID, setEmployeeHID] = useState(null);
+
   // Make a GET request to API endpoint for the EmployeeHID of the logged in employee
   useEffect(() => {
     const fetchEmployeeHID = async () => {
       try {
-        await axios.post(
-          `${config.apiUrl}/SubmissionForm/SetSelectedEmployeeHID`,
-          -1,
+         const response = await axios.get(
+          `${config.apiUrl}/SubmissionForm/GetEmployeeHID`,
           {
             headers: {
               "Content-Type": "application/json",
             },
           }
         );
+
+        setEmployeeHID(response.data);
+
       } catch (error: any) {
         console.error("Error fetching data:", error.message);
       }

```

Comment:
move this GET request up to the parent component (App.tsx) where i have loggedHID and set that state with this